lib/src/context.dart

```diff
@@ -94,6 +95,15 @@ class Context {
   static const int showEmailsAcceptedContacts = 1;
   static const int showEmailsAll = 2;
 
+  static const int DC_MSG_UNDEFINED = 0;
```

Comment:
Please adjust the naming in regards to the Dart language guidelines (DC_MSG_TEXT in java would be msgText in Dart).